perf/opf/opf.jl

```diff
@@ -0,0 +1,30 @@
+ROSETTA_OPF_DIR = "/home/blegat/git/rosetta-opf"
+include(joinpath(ROSETTA_OPF_DIR, "jump.jl"))
+model = opf_model(parse_data(joinpath(ROSETTA_OPF_DIR, "data/opf_warmup.m")))
```

Comment:
It's not useful to test this one. You really need a much larger file to stress test the code paths.